core/SellCompany.java

```diff
@@ -0,0 +1,87 @@
+package edu.asu.c3simulator.widgets;
+
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.g2d.BitmapFont;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton.TextButtonStyle;
+import com.badlogic.gdx.scenes.scene2d.ui.Window;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+
+/**
+ * Creates a button that in turn creates a popup window when clicked. The popup
+ * window requires confirmation via button click before selling the company.
+ * 
+ * @author Justin
+ *
+ */
+public class SellCompany extends Table
+{
+
+	private TextButton button;
+	private TextButtonStyle textButtonStyle;
+	private BitmapFont font;
+	private Window window;
+
+	public SellCompany(final Game game, final Skin skin)
+	{
+		super();
+		final TextButton continueButton = new TextButton("Continue", skin);
+		continueButton.addListener(new ClickListener() {
+			@Override
+			public void clicked(InputEvent event, float x, float y)
+			{
+				System.out.println("Continue!");
+				// TODO: Delete actual business
+			}
+		});
+
+		final TextButton cancelButton = new TextButton("Cancel", skin);
+		cancelButton.addListener(new ClickListener() {
+			@Override
+			public void clicked(InputEvent event, float x, float y)
+			{
+				System.out.println("Cancel.");
```

Comment:
These statements should be left out of production code unless they are placeholders. If you want notifications, use a logger, such as the ApacheLogger declared in our spec.